blocks/byline-block/features/byline/index.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
```

Comment:
This file needs renamed back to be `default.jsx`

Reply:
I thought I did this and just now realized that my push did not go through because I was on a sub-branch.  :doh:

Pushing now.